5-infrastructure/modules/cloud-sql/vpc-connection.tf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_service_networking_connection" "private_vpc_connection" {
```

Comment:
Any reason not to use https://github.com/terraform-google-modules/terraform-google-sql-db/tree/master/modules/private_service_access ?

Reply:
the only difference is setting a cusom name on the global address, and as the module is called 2 times it was making 4 resources instead of just 2 if I simply call the CFT module once in base-env
Please let me know how to approach, I have made the change to the CFT module as it seems correct, can revert it back if needed

Reply:
Please see comment and update in df7ee2b0ae3063ec554e274365595cf4d6f97f3d